src/snowflake/cli/api/project/util.py

```diff
@@ -42,12 +42,20 @@ def encode_uri_component(s: str) -> str:
     return quote(s, safe="!~*'()")
 
 
-def clean_identifier(input_: str):
+def sanitize_identifier(input_: str):
```

Comment:
odd that we didn't have tests for this utility, do you mind adding some? I know you cover it through templates, but there should be some standalone tests too.

Reply:
sure i can do that.